components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/AuthorizationValve.java

```diff
@@ -70,7 +70,8 @@ public void invoke(Request request, Response response) throws IOException, Servl
             }
 
             String requestURI = request.getRequestURI();
-            if (requestURI.startsWith("/o/")) {
+            String organizationPathParam = "/o/";
```

Comment:
Shouldn't this be a constant, rather than a variable?

Reply:
fixed in [72666b8](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/189/commits/72666b88da1af99aac78df2f32ae82f5d6ab8dc9)